### PR TITLE
Fetch ice credentials once per session or if they are expired

### LINKED ISSUE #186: Only fetch ice servers once per session rather per test.
**Description**
It's rather wasteful to fetch new ice servers and credentials for each test. We should be able to fetch them once and reuse them for all tests.

cc @andresusanopinto

### PR DESCRIPTION
**Pull request by KaptenJansson**
**Description**
Fixes #186 

**Purpose**

### CODE REVIEW

File: src/js/call.js [hunk 1]
```diff
@@ -152,7 +152,7 @@ Call.parseCandidate = function(text) {
   };
 };
```
Comment (andresusanopinto) [inline src/js/call.js]: Define "Call.origResponse_ = null" here and comment what the static variable does.
Btw could you get a more descriptive name? Call.cachedIceServers_?

Comment (KaptenJansson) [inline src/js/call.js]: Done!

File: src/js/call.js [hunk 2]
```diff
@@ -206,7 +220,14 @@ Call.fetchTurnConfig_ = function(onSuccess, onError) {
     }
 
     var response = JSON.parse(xhr.responseText);
-    onSuccess(response);
+    Call.cachedIceServers_ = response;
+    Call.getCachedIceCredentials_ = function() {
+      // Make a new object due to tests modifying the original response object.
+      return JSON.parse(JSON.stringify(Call.cachedIceServers_));
+    };
+    Call.fetchTurnTime_ = Date.now();
```
Comment (andresusanopinto) [inline src/js/call.js]: Declare this variable next to Call.cachedTurnConfig. Maybe name it "Call.cachedTurnConfigTime_"

Comment (andresusanopinto) [inline src/js/call.js]: Better yet... compute here the expireTime and call it:
"Call.cachedTurnConfigExpireTime_"

Comment (KaptenJansson) [inline src/js/call.js]: Ah missed that one.

**Comment by andresusanopinto** [comment]
lgtm

**Pull request merged**